tests/MongoDB.Driver.Core.Tests/Core/Operations/EndTransactionOperationTests.cs

```diff
@@ -96,14 +99,19 @@ public void CommandName_should_return_expected_result()
         [Theory]
         [ParameterAttributeData]
         public void CreateCommand_should_return_expected_result(
+            [Values(true, false)] bool useRecoveryToken,
             [Values(1, 2)] int w)
         {
             var writeConcern = new WriteConcern(w);
-            var subject = CreateSubject(writeConcern: writeConcern);
+            var recoveryToken = BsonDocument.Parse("{ cake: false }");
+            var subject = useRecoveryToken ? CreateSubject(recoveryToken, writeConcern) : CreateSubject(writeConcern: writeConcern);
```

Comment:
There is no need to call two different overloads of CreateSubject. Instead it is simpler to just set the value of the recoveryToken variable appropriately:

    var recoveryToken = useRecoveryToken ? new BsonDocument("cake", "false") : null;
    var subject = CreateSubject(recoveryToken, writeConcern);

Reply:
Fixed